lisa/platform_.py

```diff
@@ -121,6 +122,11 @@ def get_environment_information(self, environment: Environment) -> Dict[str, str
             information.update(
                 self._get_environment_information(environment=environment)
             )
+        except ReleaseEndOfLifeException as identifier:
+            self._log.exception(
```

Comment:
It doesn't need to log out, just raise it. I don't remember if which one can keep the call trace shorter, please pick it.

```python
except ReleaseEndOfLifeException:
    raise

except ReleaseEndOfLifeException as e:
    raise e
```